content/en/docs/setup/production-environment/tools/kubeadm/control-plane-flags.md

```diff
@@ -79,8 +79,14 @@ kubernetesVersion: v1.16.0
 scheduler:
   extraArgs:
     bind-address: 0.0.0.0
-    config: /home/johndoe/schedconfig.yaml
-    kubeconfig: /home/johndoe/kubeconfig.yaml
+    config: /etc/kubernetes/scheduler-config.yaml
+    feature-gates: AllAlpha=false,AllBeta=true
```

Comment:
can we include the `config` line only; remove `feature-gates` and `bind-address`?